fiftyone/core/ontology.py

```
@@ -193,3 +272,99 @@ def from_dict(cls, d: dict) -> "AnnotationOntology":
+    root = doc.root or {}
+    instance = cls.__new__(cls)
+    Ontology.__init__(instance, name=doc.name, description=doc.description)
+    instance._doc = doc
+    instance._apply_doc(doc)
+    return instance
+
+
+# ---- Module-level CRUD functions ------------------------------------------
+
+
+def _objects_by_slug(name: str):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_objects_by_slug`

(ANN202)